src/providers/restaurant-service.ts

```diff
@@ -41,13 +43,31 @@ export class RestaurantService {
   resetLocalStorage(){
     this.storage.clear();
   }
-  getComments(restName):Promise<Comment[]>{
-    let commentList=[];
+  getComments(restName){
+    return this.storage.get(restName);
+  }
+  updateComments(restName):Promise<Comment[]>{
+    this.cleanArray();
     for(let i=0; i<COMMENTS.length; i++){
       if(COMMENTS[i].restName === restName){
-        commentList.push(COMMENTS[i]);
+        this.commentList.push(COMMENTS[i]);
       }
     }
-    return Promise.resolve(commentList);
+    return Promise.resolve(this.commentList);
+  }
+  saveNewComment(item):boolean{
+    this.commentList = item;
+    this.saveComment(this.commentList[0].restName);
+    return true;
+  }
+  saveComment(name){
+    let newComment = JSON.stringify(this.commentList);
+    this.storage.set(name, newComment);
+  }
+  cleanArray(){
+    let size = this.commentList.length;
+    for(let i=0; i<size; i++){
+      this.commentList.pop();
+    }
```

Comment:
is this the best approach to clean an array? it works yeah but what if you make something like: this.commentList = nil or this.commentList = [] to avoid moving through the entire array

or some other option for iONIC2